editor/editor_node.cpp

```diff
@@ -5187,12 +5187,6 @@ EditorNode::EditorNode() {
 	top_region->add_child(left_menu_hb);
 	menu_hb->add_child(top_region);
 
-	{
-		Control *sp = memnew(Control);
-		sp->set_custom_minimum_size(Size2(30, 0) * EDSCALE);
```

Comment:
This looks like it was used to force some min height to the `menu_hb`, is there no visual difference without it? (including when resizing the window)

Reply:
Nope, there's already a `Control` doing the spacing job right next to it.